### ISSUE TITLE
Useful for managing and enhancing virtual raw ps2 hdd on PCSX2 

### ISSUE THREAD

**Issue by BenjiBenji69** [GDX-X/PFS-BatchKit-Manager#50]
This is not exactly an issue or problem, just as a reminder for the devs for future updates, that there is functionality for this

I can use programs like OSFMOUNT, mount it as physical drive, mount it as hdd and detect the virtual mounted hdd inside the program. Things or function that the old Winhiip or any other similiar program cannot do. Whats great is that it doesnt break the functionality for the dynamic expanded raw virtual hdd that pcsx2 or windows created if already formatted, which is good since im trying to find a way to install HDDOSD without destroying the dynamic partition of the virtual hdd from PCSX2 and retaining the hdd size (as if they're 100gb when in reality its dynamic expanding vhd). 

Anyway i suggest putting suggestive links for the files required, since for some reason you still need browsing around some forums to find some of these files, its literally buried. Would be helpful if it have such thing.

![image](https://github.com/GDX-X/PFS-BatchKit-Manager/assets/99275631/805f201e-e48c-497b-8722-9f6bda2677cc)

**Comment by GDX-X** [comment]
Yes it's interesting but you can create a virtual disk on windws VHD of 10GB and install the hdd-osd on it then rename the VHD to RAW so that it can be used with pcsx2

**Comment by BenjiBenji69** [comment]
yes good to know that its always intended to work with this use case, this is my first time trying the program and works just about as seamless, but thats the thing im still not sure about formatting there and im still used to just open ulaunch and run hdd manager to format it, and its probably bios issue but i dont think this method works most of the time unless the kelf is decrypted like the one i found on archive.org that has that and boot on most bios version

I think all these janky setup can be avoided, if the programs have option features  like partition cloning for ps2 format hdd or like WinHiip to recover partition directly when using it after using like hddrawcopytool, though obviously wont work with dynamic vhd but it would help smooth the experience

Anyway theres about opportunity for more here, i think with pcsx2 you can literally add a bunch more to customize everything with the hdd, with real ps2 theres some limitation i think but not with this, i'll suggest something more later
